### PR TITLE
23 modified files from github_advisory_sync.rb script

### PR DESCRIPTION
**Pull request by jasnow**
23 modified (2013) files from github_advisory_sync.rb script

Straight from github_advisory_sync.rb script.
Then I manually reviewed them and fixes title/description fields (tried to be consistent).
Ran "yamllint" (fixes issues).
Ran "rake" (clean).

This is 1st set of 3 modified 2013 files.

### CODE REVIEW

File: gems/actionpack/CVE-2013-4491.yml [hunk 1]
```diff
@@ -3,17 +3,17 @@ gem: actionpack
...
 osvdb: 100528
+ghsa: 699m-mcjm-9cw8
 url: https://groups.google.com/forum/#!topic/ruby-security-ann/pLrh6DUw998
 title: Reflective XSS Vulnerability in Ruby on Rails
 date: 2013-12-03
 description: |
   There is a vulnerability in the internationalization component of Ruby on
   Rails. Under certain common configurations an attacker can provide specially
   crafted input which will execute a reflective XSS attack.
-
```
Comment (postmodern) [inline gems/actionpack/CVE-2013-4491.yml:13]: Did `yamllint` flag this? The `|` modifier allows for multiple paragraphs along as they are indented.

File: gems/bundler/CVE-2013-0334.yml [hunk 2]
```diff
@@ -2,11 +2,12 @@
 gem: bundler
 cve: 2013-0334
 osvdb: 110004
+ghsa: 49jx-9cmc-xjxm
 url: https://nvd.nist.gov/vuln/detail/CVE-2013-0334
-title: "CVE-2013-0334 rubygem-bundler: 'bundle install' may install a gem from a source\
-  \ other than expected"
+title: 'CVE-2013-0334 rubygem-bundler: ''bundle install'' may install a gem from a source other than expected'
```
Comment (postmodern) [inline gems/bundler/CVE-2013-0334.yml:7]: Maybe we should use double-quotes if the string contains single-quotes? Your call.

**Review by postmodern — COMMENTED** [review]
Only two things stood out.

File: gems/actionpack/CVE-2013-4491.yml (hunk 1 above)
Comment (jasnow) [reply gems/actionpack/CVE-2013-4491.yml:13]: Are you commenting on the blank line that was removed?

**Comment by jasnow** [comment]
All checks are green after changes.

**Review by postmodern — CHANGES_REQUESTED** [review]

File: gems/bundler/CVE-2013-0334.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ cve: 2013-0334
 osvdb: 110004
 ghsa: 49jx-9cmc-xjxm
 url: https://nvd.nist.gov/vuln/detail/CVE-2013-0334
-title: 'CVE-2013-0334 rubygem-bundler: ''bundle install'' may install a gem from a source other than expected'
+title: 'CVE-2013-0334 rubygem-bundler: "bundle install" may install a gem from a source other than expected'
```
[the remark this answers is not in the document]
Comment (jasnow) [reply gems/bundler/CVE-2013-0334.yml:7]: That is done a lot in the database: git grep "'" |grep title

[the remark this answers is not in the document]
Comment (jasnow) [reply gems/bundler/CVE-2013-0334.yml:7]: 98 times

**Pull request merged**